codespell_lib/data/dictionary.txt

```diff
@@ -10264,15 +10294,33 @@ enclosue->enclosure
 enclosung->enclosing
 enclude->include
 encluding->including
+encocde->encode
 encocded->encoded
+encocder->encoder
+encocders->encoders
+encocdes->encodes
+encocding->encoding
+encocdings->encodings
+enconde->encode
+enconded->encoded
+enconder->encoder
+enconders->encoders
+encondes->encodes
 enconding->encoding
 encondings->encodings
+encorde->encode, encore,
 encorded->encoded
+encorder->encoder
+encorders->encoders
+encordes->encodes
+encording->encoding
+encordings->encodings
 encorporating->incorporating
-encose->enclose
-encosed->enclosed
-encoses->encloses
-encosing->enclosing
+encose->enclose, encode,
+encosed->enclosed, encoded,
+encoses->encloses, encodes,
```

Comment:
Can we add:
encoser->encoder
encosers->encoders

You could also suggest encloser(s), but that seems to be fairly rare.

Reply:
8f0960943d8c462b58cbce05241c261c365de429